AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/Events/SyncQueriesStartedEvent.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Used as HubPayload for the `SyncQueriesStarted`
+public struct SyncQueriesStartedEvent {
+    public let models: [String] /// list of model names
```

Comment:
triple `///` is used for documentation i believe, i don't think putting it on the right side will pick it up